src/self_update.rs

```diff
@@ -49,7 +49,7 @@ pub fn self_update() -> Result<()> {
 
             #[cfg(windows)]
             {
-                let status = command.spawn().and_then(|mut c| c.wait())?;
+                let status = command.status_checked()?;
```

Comment:
Lots of replacements like this, variants of `spawn()?.wait()?` replaced with `.status_checked()?`.